libcaf_test/caf/test/unit_test.hpp

```diff
@@ -36,6 +36,15 @@ class message;
 namespace test {
 
 /**
+ * Default test-running function.
+ * This function will be called automatically unless you define
+ * @c CAF_TEST_NO_MAIN before including "caf/test/unit_test.hpp". In
```

Comment:
Nit: Doxygen supports Markdown natively, just wrap the code-formatting in backticks.